<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolsDialog</class>
 <widget class="QDialog" name="ToolsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelTxId">
         <property name="text">
          <string>transaction id: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QValidatedLineEdit" name="txidEdit">
         <property name="toolTip">
          <string>The transaction id</string>
         </property>
         <property name="placeholderText">
          <string notr="true">Enter a transaction id...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="verboseCheckbox">
         <property name="toolTip">
          <string>verbose</string>
         </property>
         <property name="text">
          <string>verbose</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelBlockHash">
         <property name="text">
          <string>blockhash (optional): </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QValidatedLineEdit" name="blockHashEdit">
         <property name="toolTip">
          <string>The block in which to look for the transaction</string>
         </property>
         <property name="placeholderText">
          <string notr="true">Enter Blockhash...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="helpMessage">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>4</width>
         <height>4</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="submitButton">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qt/qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <connections>
  <connection>
   <sender>submitButton</sender>
   <signal>accepted()</signal>
   <receiver>ToolsDialog</receiver>
   <slot>onSubmitForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitButton</sender>
   <signal>rejected()</signal>
   <receiver>ToolsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
